python/llm/src/bigdl/llm/transformers/model.py

```diff
@@ -105,6 +107,19 @@ def optimize(cls, model):
             # todo implement 4.28.0 ~ 4.30.2
             pass
 
+        if "chatglm2" in model.config._name_or_path:
+            modeling_module_name = model.__class__.__module__
+            module = importlib.import_module(modeling_module_name)
+            from bigdl.llm.transformers.models.chatglm import chatglm_attention_forward_8eb45c
+            from bigdl.llm.transformers.models.chatglm import cross_attn_forward_8eb45c
+            convert_forward(model,
+                            module.SelfAttention,
+                            chatglm_attention_forward_8eb45c
+                            )
+            convert_forward(model,
+                            module.CoreAttention,
+                            cross_attn_forward_8eb45c)
+
```

Comment:
we also need to call optimize in `load_low_bit`? Alternatively, we may refactor `load_convert` and `ggml_convert_quant` so that `load_pretrained` and `load_low_bit` call the same underlying method